web/src/App.tsx

```diff
@@ -11,7 +11,12 @@ import Routes from 'src/Routes'
 
 import './index.css'
 
-const extendedTheme = extendTheme(theme)
+export const colors = {
+  brand: {
+    gray: 'gainsboro',
+  },
+}
```

Comment:
This should have gone in the `config/chakra.config.ts` file. Now, the theme defined there is not taken into account.

Reply:
Okay I change it in the new branch